app/models/user.rb

```diff
@@ -3,7 +3,6 @@
 class User < ApplicationRecord
   before_save :set_api_key
   nilify_blanks
-  validates :email, presence: true
```

Comment:
I think `:contactable` can be validated with something like:
`validates :contactable, inclusion: { in: [true, false] }`